docs/guides/deployment/deploy-to-fly-io.md

```diff
@@ -1,14 +1,6 @@
-# Deployment
+# Deploy to Fly.io
 
-Requirements:
-
-1. Dockerfile with access to `platformatic` CLI
-2. A fly.io account
-3. A platformatic app that works locally
-
-## On Fly.io
-
-1. Need a fly.io account and the CLI tool: https://fly.io/docs/hands-on/
+1. Need a Fly.io account and the CLI tool: https://fly.io/docs/hands-on/
 2. Navigate to your project on your local machine
 1. Create a **Dockerfile**:
```

Comment:
Is there any benefit to deploying to Fly with a custom Dockerfile? I think it'll be much simpler letting Fly auto-detect everything.

Reply:
For simple cases, there is no benefit. It is significantly simpler to have a `"start": "platformatic db"` script in **package.json** and let Fly do its work.

If the user wants additional features, like backups with Litestream, the Dockerfile becomes necessary.